File: 0344-reverse-string/0344-reverse-string.cpp
class Solution {
public:
    void reverseString(vector<char>& s) {
        int n = s.size();
        int l = 0;
        int r = n-1;
        while(l<=r){
            int x = s[l];
            s[l] = s[r];
            s[r] = x;

            l++;
            r--;
        }
    }
};

File: 0344-reverse-string/0344-reverse-string.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        n = len(s)
        p1 = 0
        p2 = n-1

        while(p1<=p2):
            x = s[p1]
            s[p1] = s[p2]
            s[p2] = x

            p1+=1
            p2-=1
        
        